server/mysterium_api_fake.go

```diff
@@ -97,7 +97,7 @@ func (client *ClientFake) FindProposals(providerID string) (proposals []dto_disc
 
 // SendSessionStats heartbeats that session is still active + session upload and download amounts
 func (client *ClientFake) SendSessionStats(sessionId session.SessionID, sessionStats dto.SessionStats, signer identity.Signer) (err error) {
-	log.Info(mysteriumAPILogPrefix, "Session stats sent: ", sessionId)
+	log.Info(mysteriumAPILogPrefix, "SessionDto stats sent: ", sessionId)
```

Comment:
Is it correct that `SessionDto` should be here?

Reply:
Fixed